Monika After Story/game/script-fun-facts.rpy

```diff
@@ -757,3 +757,30 @@ label mas_fun_fact_cloud_weight:
     #Call the end
     call mas_fun_facts_end
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_fun_facts_database,
+            eventlabel="mas_fun_fact_coffeelegends",
+        ),
+        code="FFF"
+    )
+
+label mas_fun_fact_coffeelegends:
+    m 3eub "The last time I had a cup of coffee, I got a little curious about where it was first discovered."
+    m 3euc "You see, the use of coffee is recorded consistently around the 15th century, but...{w=0.2}it's unclear {i}how{/i} exactly it was found. There are quite a few legends claiming to be the very first."
+    m 1eua "Several accounts involve farmers or monks observing animals acting oddly after eating some strange, bitter berries."
+    m 1hub "When they tried them for themselves, they were startled to find themselves energized as well!"
+    m 2euc "One such myth claims that an Ethiopian monk named Kaldi brought the berries to a nearby monastery, wanting to share what he'd found.{w=0.1}{nw}"
+    extend 2eksdld "But when he did so, he was met with disapproval, and the coffee beans were thrown into a fire."
+    m 1dub "As they burned, however...the beans began to let off the most {i}delicious{/i} aroma. It was so enticing that the monks scrambled to save the beans and put them in water."
+    m 1dua "Thus producing the first cup of coffee."
+    m 2euc "Another claim states that an Islamic scholar named Omar discovered coffee beans during his exile from Mecca. {nw}"
+    extend 2ekd "At the time, he was starving and struggling to survive. If not for the energy they provided, he may have died."
+    m 1eub "But when word of his discovery spread, Omar was asked to return and was made a saint."
+    m 1esa "Whether or not that was truly its first use, coffee became very prevalent in the Islamic world after its discovery.{w=0.2} For example, during fasting periods it was used to ease hunger and remain energized."
```

Comment:
```suggestion
    m 1esa "Whether or not that was truly its first use, coffee became very prevalent in the Islamic world after its discovery.{w=0.2} For example, during fasting periods it was used to ease hunger and help people remain energized."
```
This is a subtle grammar thing. The way the verb is used in `used to ease hunger` and `used to remain energized` are semantically different and incompatible. You can't connect them like that, you need some kind of a buffer predicate.